### ISSUE TITLE
Unable to Build Package

### ISSUE THREAD

**Issue by Peyton-McKee** [aws-amplify/amplify-ios-maplibre#28]
### Describe the bug

When adding the package to my project and trying to build, I get the error ```Cannot find type 'CredentialsProvider' in scope``` in AmplifyLibreMapAdapter/Sources/AmplifyLibreMapAdapter/AWSMapURLProtocol+GeoConfig 

### Steps To Reproduce

```swift
Steps to reproduce the behavior:
1. Go to 'Add new package’
2. Add Package with both package components
3. Build
4. See Error
```

### Expected behavior

It should build 

### Amplify MapLibre Adapter Version

2.0.1

### Swift version

5

### Xcode version

15.1

### Is this a regression?

No

### Device

N/A

### iOS Version

IOS 17

### Additional context

When I right click on credentialsProvider it takes me to the class. So Not really sure whats going on there. I also tried installing amplify first and then installing this package and the same thing happened.

**Comment by Peyton-McKee** [comment]
Ive tried just creating a brand new project and same thing happens.

**Comment by Peyton-McKee** [comment]
I think its a bug... I manually added the code from scratch into my project and changed it to credentialsproviding to actually be the protocol and it worked

**Comment by atierian** [comment]
Thanks for reporting this @Peyton-McKee.
This is due a change to a type name in the AWS SDK for Swift. We're working on a fix -- in the meantime, you can pin to Amplify Swift 2.2.0

**Comment by ruisebas** [comment]
Fixed in [2.1.0](https://github.com/aws-amplify/amplify-ios-maplibre/releases/tag/2.1.0)
